client/task-list/tasks/products/product-template-modal.js

```diff
@@ -84,6 +98,19 @@ export default function ProductTemplateModal( { onClose } ) {
 		}
 	};
 
+	if ( window.wcAdminFeatures && ! window.wcAdminFeatures.subscriptions ) {
```

Comment:
I thought this should also happen if `profileItems.product_types.includes('subscriptions') && installedExtensions.includes('woocommerce-payments')`?
At the moment the `subscription product` option shows up always when the `subscriptions` feature flag is enabled.

Reply:
Nice catch! I fixed this in the commit 583cf8a